src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.CommonSVTControls.ps1

```diff
@@ -790,6 +793,7 @@ class CommonSVTControls: ADOSVTBase {
                     }
 
                     $groups = $secureFileWithBroaderGroup | ForEach-Object { $_.Name + ': ' + $_.Role } 
+                    $controlResult.AdditionalInfo = "Count of broader groups that have user/administrator access to secure file: $($secureFileWithBroaderGroupCount)"
```

Comment:
not consistent with the changes in feeds broader grp control. (list of broader grps not added in additionalinfo here, but is it added in the former control)